### ISSUE TITLE
False warning with puppet-lint-0.2

### ISSUE THREAD

**Issue by alvagante** [rodjek/puppet-lint#116]
[root@pe-puppet modules]# rvm gemset use puppet-lint
Using /usr/local/rvm/gems/ruby-1.8.7-head with gemset puppet-lint
[root@pe-puppet modules]# puppet-lint foo/manifests/params.pp

[root@pe-puppet modules]# rvm gemset use puppet-lint-2
Using /usr/local/rvm/gems/ruby-1.8.7-head with gemset puppet-lint-2
[root@pe-puppet modules]# puppet-lint foo/manifests/params.pp 
WARNING: unquoted resource title on line 64

The involved line(s)

```
  $config_file_init = $::operatingsystem ? {
    /(?i:Debian|Ubuntu|Mint)/ => '/etc/default/foo',
    default                   => '/etc/sysconfig/foo',
  }
```
